ako-operator/bundle/manifests/ako.vmware.com_akoconfigs.yaml

```diff
@@ -109,6 +109,11 @@ spec:
                     description: ServicesAPI enables AKO to do Layer 4 loadbalancing
                       using Services API
                     type: boolean
+                  useDefaultSecretsOnly:
```

Comment:
we must cherry-pick these changes to master. I am seeing these changes in every release.